checkov/secrets/scan_git_history.py

```diff
@@ -54,17 +64,16 @@ def scan_history(self, last_commit_scanned: Optional[str] = '') -> bool:
         return scanned
 
     def _scan_history(self, last_commit_scanned: Optional[str] = '') -> bool:
-        # commits_diff = self._get_commits_diff(self.root_folder, last_commit_sha=last_commit_scanned)
-        self.commits_diff = self._get_commits_diff(self.root_folder, last_commit_sha=last_commit_scanned)
-        if not self.commits_diff:
-            return False
-        logging.info(f"[_scan_history] got {len(self.commits_diff)} files diffs in {self.commits_count} commits")
+        commits_diff: List[Commit] = []
+        commits_diff.extend(self._get_first_commit())
+        commits_diff.extend(self._get_commits_diff(last_commit_sha=last_commit_scanned))
```

Comment:
a more pythonic way to generate this:
```suggestion
        commits_diff: List[Commit] = [*self._get_first_commit(), *self._get_commits_diff(last_commit_sha=last_commit_scanned)]
```